### PR TITLE
Wars

### PR DESCRIPTION
**Pull request by Idhrendur**

### CODE REVIEW

File: HoI4Mod/Rome_Strikes_Back/common/decisions/lar_agent_recruitment_decisions.txt [hunk 1]
```diff
@@ -696,7 +696,7 @@ lar_local_recruitment = {
 			FROM = {
 				is_on_continent = asia
 				NOT = {
-					OWNER = { has_country_flag = conv_south_asia }
```
Comment (CaesarVincens) [inline HoI4Mod/Rome_Strikes_Back/common/decisions/lar_agent_recruitment_decisions.txt:699]: The scope on these was causing issues. As it's checking a country flag on a state, so the owner scope change is needed.

Comment (Idhrendur) [reply HoI4Mod/Rome_Strikes_Back/common/decisions/lar_agent_recruitment_decisions.txt:699]: Ah, shoot. I got mixed up and thought this was a converter change that had been made in the meanwhile.

File: HoI4Mod/Rome_Strikes_Back/common/decisions/categories/00_decision_categories.txt [hunk 2]
```diff
@@ -227,6 +227,13 @@ lar_local_recruitment = {
 	}
 }
 
+ENG_imperial_conference_decision = {
```
Comment (CaesarVincens) [inline HoI4Mod/Rome_Strikes_Back/common/decisions/categories/00_decision_categories.txt:230]: Why are the imperial conference decisions being added (back) in?

Comment (Idhrendur) [reply HoI4Mod/Rome_Strikes_Back/common/decisions/categories/00_decision_categories.txt:230]: I know they were added to the converter recently, but if you had purposefully removed here I hadn't realized.

Comment (CaesarVincens) [reply HoI4Mod/Rome_Strikes_Back/common/decisions/categories/00_decision_categories.txt:230]: No, I didn't remove them, I'm just curious why they'd be added when England doesn't have an empire.

**Pull request merged**